heron/stmgr/src/cpp/manager/checkpoint-gateway.h

```diff
@@ -85,7 +89,7 @@ class CheckpointGateway {
     std::deque<Tuple> ForceDrain();
     void Clear();
    private:
-    void add(Tuple _tuple, sp_uint64 _size);
+    void add(Tuple& _tuple, sp_uint64 _size);
     void add_front(Tuple _tuple, sp_uint64 _size);
```

Comment:
maybe better to be "Tuple&" too here like the function above?

Reply:
fixed